Sources/Crust/Parser.swift

```diff
@@ -30,6 +30,19 @@ extension Diagnostic.Message {
   static func expected(_ name: String) -> Diagnostic.Message {
     return Diagnostic.Message(.error, "expected \(name)")
   }
+
+  static func expectNameInFuncDecl(_ syntax: ExprSyntax) -> Diagnostic.Message {
+    return Diagnostic.Message(.error,
+                              "unexpected expression '\(syntax.sourceText)'")
```

Comment:
This needs a better message

Reply:
> expression '<expr>' may not be used as identifier in function name

?

Reply:
That seems good to me.

Reply:
I don't think we should put the source text in there -- I think we could go further with reprinting expressions that are highlighted instead.